tensorflow/compiler/xla/service/gpu/ir_emitter_nested.cc

```diff
@@ -38,7 +38,7 @@ namespace gpu {
 IrEmitterNested::IrEmitterNested(const HloModuleConfig& hlo_module_config,
                                  const HloComputation& nested_computation,
                                  IrEmitterContext* ir_emitter_context)
-    : IrEmitter(hlo_module_config, ir_emitter_context, /*is_nested=*/true) {
+    : IrEmitter(hlo_module_config, ir_emitter_context, /*is_nested=*/true, nullptr) {
```

Comment:
Please use style `/*foo=*/nullptr`.  We have checks that ensure you use the right `foo` here.